plugin/awesoMD/plugin.js

```diff
@@ -454,6 +462,34 @@ const Plugin = () => {
 		return [content, options];
 	}
 
+	/**
+	 * Add slide separator in case where the heading indicates the start of new slide
+	 *
+	 * Returns the updated markdown file with added slide separator above every slide headings
+	 */
+	function addSlideSeparator(markdown) {
+		const defaultSlideSeparator = '---';
```

Comment:
we should not leave the separator hard-codded, but should make it a parameter to the function and pass it in from `options.separator`